### DISCUSSION TITLE
Icon does show up in pdf ?

### DISCUSSION THREAD

**Discussion by Bloe1901** [FlorianWoelki/obsidian-icon-folder#604] [Q&A]
This kind of took me by surprise. Upon exporting as pdf, the icons do render and are just not text. I do remember few weeks ago when render, it just output text, and I haven't update since then (2.14.3). I have checked the issues tab and #335 is still open, as the task in projects. So, I am asking if you guys have implemented that somehow? or this is just weird behavior. I do not want to create a new issue since I don't know if this count as a bug or not. Hope I get an answer, thank you !

The pdf exported with icons: 
[Plane.pdf](https://github.com/user-attachments/files/17599587/Plane.pdf)

The only pdf I kept that exported without icons 
[Plane (1).pdf](https://github.com/user-attachments/files/17599619/Plane.1.pdf)

**Comment by FlorianWoelki** [comment]
Hey there, currently, it is not implemented anywhere. Feel free to watch the issue to check the status of the issue. I have currently limited time to work on this project.
